test/system/005-info.bats

```diff
@@ -88,6 +88,18 @@ host.slirp4netns.executable | $expr_path
     is "$output" ".*graphOptions: {}" "output includes graphOptions: {}"
 }
 
+@test "podman info netavark " {
+    # Confirm podman uses netavark only when requested
```

Comment:
Careful with this "only", it is misleading.
* If you really truly mean "only", then you should do the converse test: in the `else` clause, confirm that `is_netavark` is false.
* Otherwise, please remove the `only`. That makes it clearer that it's OK to use netavark even if `NETWORK_BACKEND` is not set or if it's set to something else.